deps/picotls/t/util.h

```diff
@@ -124,10 +124,27 @@ static inline void setup_session_file(ptls_context_t *ctx, ptls_handshake_proper
     }
 }
 
-static inline void setup_verify_certificate(ptls_context_t *ctx)
+static X509_STORE * openssl_init_cert_store(char const * crt_file)
```

Comment:
```suggestion
static inline X509_STORE* init_cert_store(char const *crt_file)
```

This function looks good, all the points below are editorial:
* This function is defined in a header file and therefore be marked as static and inline to avoid compiler generating multiple external symbols and to suppress "unused function" warning.
* Identation and insertion of white spaces should follow that enforced by `clang-format`.
* We might not need the "openssl_" prefix considering the fact that this function is a sub-function of `setup_verify_certificate` that only deals with the OpenSSL backend without having the prefix.

Reply:
updated based on your suggestion.